ztp/gitops-subscriptions/argocd/deployment/resource-hook-cluster-role.yaml

```diff
@@ -0,0 +1,147 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: converter
+rules:
+# Read Site resources
+- apiGroups:
+  - ran.openshift.io
+  resources:
+  - siteconfigs
+  verbs:
+  - get
+  - watch
+  - list
+  # For debug
+  - create
+  - delete
+  - patch
+  - update
+
+
+- apiGroups:
```

Comment:
I believe you can collapse these apiGroups into one:
- apiGroups:
  resources:
  - configmaps
  - namespaces
  - secrets
  ..

Reply:
Yes, but I noticed in different operator examples that this is not a common practice. Probably when it isn't collapsed it's less error prone, less of a chance to affect something unwanted when changing stuff